compiler/tests/fixtures/strings.fe

```diff
@@ -0,0 +1,3 @@
+contract Foo:
+    pub def bar(s: string[100]) -> string[100]:
```

Comment:
Just repeating what we discussed in chat: As it stands, I think the syntax is a bit confusing because we have `u256[100]` defined as "An array with 100 `u256` items" so following that logic one could easily believe `string[100]` means "an array of 100 dynamically sized strings". Maybe an alternative syntax could be `string100`. That would be pretty much in line with something like `u8`, `u16`, ...; `u256`.